package-res/reportviewer/reportviewer.js

```diff
@@ -449,13 +449,13 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
           }
 
           pc.registerPageNumberChangeCallback(function(pageNumber) {
-            this.pageChanged(this.reportPrompt.panel, pageNumber);
+            this.pageChanged(pageNumber);
           }.bind(this));
         },
 
         pageChanged: function(pageNumber) {
           this._setAcceptedPage('' + (pageNumber - 1));
-          this.reportPrompt.panel.submit({isPageChange: true});
+          this.reportPrompt.panel.submit(this.reportPrompt.panel, {isPageChange: true});
```

Comment:
Ok. I agree. Lets put it there again and later when we review the submit we can review that. Thanks for that anna